ifbo/priors/ftpfn_prior.py

```diff
@@ -496,7 +496,7 @@ def __init__(self, in_dim: int, out_dim: int) -> None:
             in_dim - 2, out_dim
         )
 
-    def forward(self, *x, **kwargs) -> torch.Tensor:
+    def forward(self, x: torch.Tensor, **kwargs: Any) -> torch.Tensor:
```

Comment:
``*x`` has to be a tuple here; otherwise, the line below will raise an exception.